addons/account_bank_statement_import/i18n/es.po

```diff
@@ -754,7 +754,7 @@ msgstr "Este archivo no contiene ningún estado de cuenta %s."
 #: code:addons/account_bank_statement_import/account_bank_statement_import.py:0
 #, python-format
 msgid "This file doesn't contain any transaction for account %s."
-msgstr "Este archivo no contiene ninguna transacción para la cuenta"
+msgstr ""
```

Comment:
```suggestion
msgstr "Este archivo no contiene ninguna transacción para la cuenta %s."
```

Reply:
@pedrobaeza this isn't a PR to be merged, just to see the problem and see what we can do about. Internal discussion about integrating this kind of test in the runbot/synchronisation bot

Reply:
Ah, OK, thanks for the clarification.